### ISSUE TITLE
Documentation for new Menu features

### ISSUE THREAD

**Issue by wldcordeiro** [pedronauck/docz#405]
In the release for the new Menu system it documents how to configure it in the rc file but mentions you can use the component as well, it'd be nice for the documentation to reflect both these things.

**Comment by wldcordeiro** [comment]
Do we still use the mdx header matter @pedronauck 

```
---
name: Setup
route: /package/setup
---
```

```js
module.exports = {
  menu: [
    {
      name: 'Pakage',
      menu: [
        {
          name: 'Setup',
          route: '/package/setup',
        },
        {
          name: 'Components',
          route: '/package/components',
        },
      ],
    }
  ],
};
```

How do I migrate to this new system and organize my pages? The old way wasn't very documented either (mostly figured out from watching the preview video on the main site), and I'm seeing weird funkiness trying to get menus to play nice and find pages.

**Comment by Chun-MingChen** [comment]
New `menu` feature seems awesome! It works fine and is able to show the menu in nested. However, how could we categorize `mdx content` into the submenu? e.g.


`alert.mdx`
```
---
name: Alert
---
```

`doczrc.js`
```
├── menu
│   ├── Home
│   └── Components
│          ├── Alert     <--- how to categorize alert.mdx into here?
│          └── Button
```

**Comment by wldcordeiro** [comment]
@Chun-MingChen indeed, that's what I'm wondering as well. I couldn't figure it out! 😆

**Comment by danielbayerlein** [comment]
#436

**Comment by Jayphen** [comment]
The release link shows one level of nesting. Is it possible to nest multiple levels?

```
| Intro
| Components
|-- Buttons
|--|-- Simple <-- nested twice
|--|-- Complex
|-- Menu
|-- Footer
```

**Comment by pedronauck** [comment]
No @Jayphen, it's not possible!

**Comment by tomorrowthief** [comment]
+1 
can we support it in the future?

**Comment by priley86** [comment]
any update on nesting multiple levels? Not sure if known workarounds exist or not? I would love something like @Jayphen mentions too.


Thanks in advance.
